src/Assignments/ProblemIframe.tsx

```diff
@@ -364,7 +364,14 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
     return (
         <>
             { loading && <Spinner animation='border' role='status'><span className='sr-only'>Loading...</span></Spinner>}
-            {error && <div>{error}</div>}
+            {error &&
```

Comment:
Why aren't we using the `useAlert` hook that we use everywhere else?
variant: danger